docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,189 @@
+
+A general guideline is to use batch inference if you need to get predictions less than once an hour and are ok with the inference time taking 10 minutes longer than real-time inference (due to compute spin-up overhead).
+
+To perform batch inference:
+
+```{.python}
+cloud_predictor.predict(
+    'test.csv',  # can be a dataframe as well and the results will be stored in s3 bucket
+    instance_type="ml.m5.2xlarge",  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+    wait=True  # Set this to False to make it unblocking call
+)
+cloud_predictor.download_predict_results(
+    job_name=None  # download the most recent finished batch inference results to your local machine. Specify the job name to download a specific batch inference job's results.
+    save_path="PATH"  # If not specified, CloudPredictor will create one.
+)
+```
+
+## Retrieve CloudPredictor Info
+
+To retrieve general info about a `CloudPredictor`
+
+```{.python}
+cloud_predictor.info()
+```
+
+It will output a dict looks similar to this:
+
+```{.python}
+{
+    'local_output_path': '/home/ubuntu/XXX/demo/AutogluonCloudPredictor/ag-20221111_174928',
+    'cloud_output_path': 's3://XXX/tabular-demo',
+    'fit_job': {'name': 'ag-cloudpredictor-1668188968-e5c3',
+    'status': 'Completed',
+    'framework_version': '0.6.1',
+    'artifact_path': 's3://XXX/tabular-demo/model/ag-cloudpredictor-1668188968-e5c3/output/model.tar.gz'},
+    'recent_transform_job': {'name': 'ag-cloudpredictor-1668189393-e95c',
+    'status': 'Completed',
+    'result_path': 's3://XXX/tabular-demo/batch_transform/2022-11-11-17-56-33-991/results/test.csv.out'},
+    'transform_jobs': ['ag-cloudpredictor-1668189393-e95c'],
+    'endpoint': 'ag-cloudpredictor-1668189208-d23b'
+}
+```
+
+## Convert the CloudPredictor to a Local AutoGluon Predictor
+You can easily convert the `CloudPredictor` you trained on SageMaker to your local machine as long as you have the same version of AutoGluon installed locally.
+
+```{.python}
+local_predictor = cloud_predictor.to_local_predictor(
+    save_path="PATH"  # If not specified, CloudPredictor will create one.
+)  # local_predictor would be a TabularPredictor
+```
+
+`to_local_predictor()` would underneath downlod the tarball, expand it to your local disk and load it as a corresponding AutoGluon predictor.
+
+## Note on Image Modality
+If your training and inference tasks involve image modality, you need to
+1. make sure your column contains **absolute paths** to the images
+2. provide argument `image_column` as the column name containing image paths to `CloudPredictor` fit/inference APIs.
+
+```{.python}
+cloud_predictor.fit(..., image_column="IMAGE_COLUMN_NAME")
+cloud_predictor.predict_real_time(..., image_column="IMAGE_COLUMN_NAME")
+cloud_predictor.predict(..., image_column="IMAGE_COLUMN_NAME")
+```
```

Comment:
This probably needs its own mini tutorial to ensure they understand how to properly create the image paths and how to format the input files, I foresee this being confusing. Can be in a future PR.

Reply:
Expanded the section to give more details